formats/bibtex/index.js

```diff
@@ -0,0 +1,22 @@
+import Format from "../../src/format.js";
+import { BibLatexParser, BibLatexExporter } from "https://esm.sh/biblatex-csl-converter/lib/index.js";
+
+export default class BibTeX extends Format {
+	static defaultOptions = {
+		processUnexpected: true,
+		processUnknown: true,
```

Comment:
Not sure we’d want to expose these options wholesale. We may want to expose the _functionality_, but with a different API. I’d not expose for now — once you expose, you need to support that forever.